indico/htdocs/sass/modules/_roombooking.scss

```diff
@@ -150,10 +150,37 @@
         @include border-radius();
         width: 370px;
     }
+
+    #booked-for-user-wrapper {
+        width: 240px;
+        display: none;
+        min-height: 0;
+
+        &.hasError {
+            @include border-all($red);
+        }
+    }
+
+    #other-user {
+        display: table-cell;
```

Comment:
Well I need to set the width of `other-user` to the width of its content, not sure how to do it with flexbox